include/z64player.h

```diff
@@ -560,15 +570,11 @@ typedef struct Player {
     /* 0x06A8 */ Actor*     unk_6A8;
     /* 0x06AC */ s8         unk_6AC;
     /* 0x06AD */ u8         unk_6AD;
-    /* 0x06AE */ u16        unk_6AE;
-    /* 0x06B0 */ s16        unk_6B0;
+    /* 0x06AE */ u16        rotsNoRestFlags; // See `NOREST_ROT_` macros. If its flag isn't set, a rot steps to a rest value.
+    /* 0x06B0 */ s16        upperLimbYawSecondary;
     /* 0x06B2 */ char       unk_6B4[0x004];
-    /* 0x06B6 */ s16        unk_6B6;
-    /* 0x06B8 */ s16        unk_6B8;
-    /* 0x06BA */ s16        unk_6BA;
-    /* 0x06BC */ s16        unk_6BC;
-    /* 0x06BE */ s16        unk_6BE;
-    /* 0x06C0 */ s16        unk_6C0;
+    /* 0x06B6 */ Vec3s      headLimbRot;
+    /* 0x06BC */ Vec3s      upperLimbRot;
```

Comment:
Is `Limb` necessary here? Which limbs are encompassed in `upperLimbRot`?

Reply:
It corresponds to `PLAYER_LIMB_UPPER`